code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -627,7 +627,6 @@
 
 	var/datum/netdata/data = new
 	data.recipient_ids = splittext(target_address, ";")
-	data.sender_id = address
```

Comment:
why? sender should include it's adress into message.To provide ability response for message.

Reply:
I changed NTNet packets to auto-set `sender_id` on send operation, unless it was set already.